### PR TITLE
Funding Source Balance Model

### PR DESCRIPTION
**Pull request by ithielnor**
Built on #8

### CODE REVIEW

**Comment by therockstorm** [comment]
Can you merge master into this?

File: Dwolla.Client/Models/Responses/BalanceResponse.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Dwolla.Client.Models.Responses
+{
+    public class BalanceResponse : BaseResponse
+    {
+        public Balance Balance { get; set; }
+        public DateTime LastUpdated { get; set; }
+    }
```
Comment (therockstorm) [inline Dwolla.Client/Models/Responses/BalanceResponse.cs]: You had no way of knowing this, but in the next week or so this endpoint will be async and return immediately with an optional `string Status`. If Status="pending", Balance and LastUpdated will be null so you can change it to `DateTime?` and add the Status if you'd like.

File: Dwolla.Client/Models/Responses/BalanceResponse.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    public class Balance
```
Comment (therockstorm) [inline Dwolla.Client/Models/Responses/BalanceResponse.cs]: This is used by other endpoints so a more generic name would be better. We call it `Money` in our server code.

**Comment by ithielnor** [comment]
This has been updated.

**Review by therockstorm — CHANGES_REQUESTED** [review]

**Comment by ithielnor** [comment]
Completed.

**Comment by ithielnor** [comment]
I'm gonna close this and combine with #10 to avoid a breaking change between the two. `Money` will be in the Models namespace instead of the Responses.

**Pull request closed without merging**